catalog/spatial/geocoding/spatial-geocoding-geocoder/src/main/java/org/codice/ddf/spatial/geocoder/endpoint/GeoCoderEndpoint.java

```diff
@@ -85,11 +86,16 @@ public Response getNearbyCities(@PathParam("wkt") String wkt) {
 
         GeoCoder geoCoder = geoCoderFactory.getService();
         NearbyLocation nearbyLocation = geoCoder.getNearbyCity(wkt);
-        JSONObject jsonObject = new JSONObject();
-        jsonObject.put("direction", nearbyLocation.getCardinalDirection());
-        jsonObject.put("distance", nearbyLocation.getDistance());
-        jsonObject.put("name", nearbyLocation.getName());
-        return Response.ok(jsonObject.toJSONString()).build();
+        if (nearbyLocation != null) {
+            JSONObject jsonObject = new JSONObject();
+            jsonObject.put("direction", nearbyLocation.getCardinalDirection());
+            jsonObject.put("distance", nearbyLocation.getDistance());
+            jsonObject.put("name", nearbyLocation.getName());
+            return Response.ok(jsonObject.toJSONString())
+                    .build();
+        } else {
+            return Response.status(HttpStatus.SC_NO_CONTENT).build();
```

Comment:
Use Response.Status.NO_CONTENT to avoid pulling apache:
http://docs.oracle.com/javaee/7/api/javax/ws/rs/core/Response.Status.html#NO_CONTENT